src/components/AppNavigation/CalendarList/CalendarListTitle.vue

```diff
@@ -0,0 +1,78 @@
+<!--
+  - @copyright Copyright (c) 2019 Georg Ehrke <oc.list@georgehrke.com>
+  - @author Georg Ehrke <oc.list@georgehrke.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<AppNavigationItem
+		class="app-navigation-entry-new-calendar"
+		:title="$t('calendar', 'New calendar')"
+		@click="toggleMenu">
+		<template slot="icon">
```

Comment:
Please do not do that!
This is the reason I wanted to have strict rules ^^'
Actions needs to go at the end of the line, not ont he icon please.

Reply:
I mean common, we're creating standards, let's not break them not even a week after they're out :sob: